kubernetes/loculus/templates/ingest-deployment.yaml

```diff
@@ -136,6 +136,21 @@ spec:
                     secretKeyRef:
                       name: service-accounts
                       key: insdcIngestUserPassword
+                - name: DB_URL
+                  valueFrom:
+                    secretKeyRef:
+                      name: database
+                      key: url
+                - name: DB_USERNAME
+                  valueFrom:
+                    secretKeyRef:
+                      name: database
+                      key: username
+                - name: DB_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      name: database
+                      key: password
```

Comment:
I don't immediately see a corresponding change in other code?

Reply:
Ah, I'm guessing this just needs a rebase or something

Reply:
this is a small bug that I only now found - the cronjob also needs these keys - they are given to ingest already